plugins/mcreator-localization/help/fr_FR/block/place_sound.md

```diff
@@ -0,0 +1 @@
+Ceci est le son joué lorsque le joueur place le bloc dans le monde.
```

Comment:
These are same for plant and block so you should refer to the block tips from plant where tips are the same so they don't duplicate. Fix this for all cases of this.

Reply:
I don't understand, they are not the same. Bloc is not equal to Plante (Block is not equal to Plant)

Reply:
Plants are blocks. I usually try to re-use tips when possible, it reduces the overhead and need to maintain these tips. You can call a plant block, and sounds act the same for both